ec/ecresource/projectresource/mocks.gen.go

```diff
@@ -0,0 +1,259 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
How do we generate this file? Obviously I'm missing something, but I didn't manage to regenerate it.

Reply:
It's [this](https://github.com/elastic/terraform-provider-ec/pull/817#discussion_r1679489861) commented out `go generate` comment